state-sync/inter-component/consensus-notifications/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "consensus-notifications"
+version = "0.1.0"
```

Comment:
I wonder what day we'll make these a major version lol

Reply:
Why not when state sync v2 lands? :D Haha.. but yeah, these versions are pretty much meaningless at this point..